### PR TITLE
Adding a new question in faq page

### PR DESCRIPTION
**Pull request by manikarthikk**
Adding a new question in faq page

### CODE REVIEW

**Review by satyarapelly — APPROVED** [review]

**Comment by kabirkhan** [comment]
This doesn't include all the updates from Rhonda.

**Comment by kabirkhan** [comment]
We need to add the question about searching from Infopedia

**Review by btelnes — APPROVED** [review]
Verified that requested changes in Rhonda's 4/12 email are present.

**Review by sdolenc — APPROVED** [review]

**Comment by eltoncarr-ms** [comment]
There are a number of diffs between what is expected and what is currently on the page.

**Review by eltoncarr-ms — CHANGES_REQUESTED** [review]
There are a number of diffs between what is expected and what is currently on the page.

Answer doesn't match requirement:
- [ ] How will I be notified of courses that are applicable to my role?
- [ ] Will I be able to discover open edX courses on Infopedia?
- [ ] Will my manager see the courses I’ve taken in InfoPedia? (possibly bad encoding)

Is the email representing the final version of the faq? If so, some existing questions need to be removed as well:
- [ ] Q. Iâ€™d like to try the Open edX experience, but no courses are available to me right now.
- [ ] Q.What do I do if I cannot access a course and I get an error saying my email address belongs to another account, and to try again with a different address?

File: courses/lms/templates/static_templates/faq.html [hunk 1]
```diff
@@ -47,23 +47,35 @@ <h1 style="text-align: left;">Frequently Asked Questions</h1>
...
-        <p>Please click on the download video button located below the video player. See screenshot.</p><br>
+    <b>Q.What do I do if I cannot access a course and I get an error saying my email address belongs to another account, and to try again with a different address?</b><br>
+	    <p> A. This is because you already created an account on the Azure Skills Initiative site using your @microsoft.com email address. You should log into the Azure site on <a href="https://openedx.microsoft.com"> https://openedx.microsoft.com</a> and change your email address under Account Settings. We recommend that you update the email address with a personal email address. After this, please try to launch your course again from MyLearning or the link you were provided.</p><br>
+	<b>Q. What do I do if I am having trouble viewing a video?</b><br>
+        <p>The video player will auto-select the best video quality option for your bandwidth. You can try a lower quality option, or you can download the video if that option is available. See screenshot.
+
+</p><br>
         <img src="/static/courses/images/download_video.png" alt="download_video_img" height="400">
```
Comment (sdolenc) [inline courses/lms/templates/static_templates/faq.html:79]: can you share the url of a fullstack deployment that has these changes? if not, swing by and we'll apply these changes to one of my fullstacks and we'll share that url to the team : )

**Review by sdolenc — CHANGES_REQUESTED** [review]
i'd like to test this in fullstack before approving. can you share the url of a fullstack deployment that has these changes? if not, swing by and we'll apply these changes to one of my fullstacks and we'll share that url to the team : )

**Comment by eltoncarr** [comment]
You can also pull the raw content, let the browser render it and do the comparisons. Right now, it seems the network is flaky…

**Review by sdolenc — APPROVED** [review]

**Review by eltoncarr-ms — APPROVED** [review]
Seems key concerns were met.

**Pull request merged**